<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dygma Raise Assembly Tests</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>9</y>
      <width>621</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="logoLabel">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>Dygma_logo_color.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="light">
        <attribute name="title">
         <string>Light control</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>40</y>
           <width>441</width>
           <height>181</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QPushButton" name="light_white">
            <property name="text">
             <string>White</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="light_red">
            <property name="text">
             <string>Red</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="light_blue">
            <property name="text">
             <string>Blue</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="light_green">
            <property name="text">
             <string>Green</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="magnet">
        <attribute name="title">
         <string>Magnet Calibrate</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>60</y>
           <width>401</width>
           <height>141</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="magnet_joined">
            <property name="text">
             <string>Keyboard is Joined</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="magnet_split">
            <property name="text">
             <string>Keyboard is Split</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <zorder>plainTextEdit</zorder>
        <zorder>gridLayoutWidget_2</zorder>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="log_messages">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDygma_Raise_Assembly_Tests">
    <property name="title">
     <string>Dygma Raise Assembly Tests</string>
    </property>
   </widget>
   <addaction name="menuDygma_Raise_Assembly_Tests"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
